packages/client/src/workspace/store.ts

```diff
@@ -175,79 +166,86 @@ async function getLandingPage(): Promise<StorefrontLandingPage> {
         title: "Explore",
         templateId: StorefrontTemplate.AppGrid,
         templateData: {
-          apps: [reactiveTraderFx, reactiveAnalytics, limitCheckerView],
+          apps: getApps(),
         },
       },
       image: {
-        src: `${BASE_URL}/images/previews/reactive-trader.PNG`,
+        src: `${WS_BASE_URL}/images/previews/reactive-trader.PNG`,
       },
     },
     topRow: {
       title: "Collections",
       items: [
         {
-          id: "top-row-item-1",
-          title: "Research",
-          description:
-            "Applications to research the current market data, trends etc...",
+          id: "collections-apps",
+          title: "Web Apps",
+          description: "A collection of web apps built using OpenFin.",
           image: {
-            src: `${BASE_URL}/images/coding-1-unsplash.jpg`,
+            src: `${WS_BASE_URL}/images/previews/reactive-trader.PNG`,
           },
           templateId: StorefrontTemplate.AppGrid,
           templateData: {
-            apps: [reactiveAnalytics],
+            apps: getApps(),
           },
         },
         {
-          id: "top-row-item-2",
-          title: "Tools",
-          description: "Tools to help day to day operations",
+          id: "collections-views",
+          title: "Views",
+          description:
+            "A collection of views made available through our catalog.",
           image: {
-            src: `${BASE_URL}/images/coding-2-unsplash.jpg`,
+            src: `${WS_BASE_URL}/images/previews/trades-view.PNG`,
           },
           templateId: StorefrontTemplate.AppGrid,
           templateData: {
-            apps: [limitCheckerView],
+            apps: getViews(),
           },
         },
       ],
     },
     middleRow: {
-      title:
-        "A collection of simple views that show how to share context using the Interop API.",
-      apps: [reactiveTraderFxTradesView, reactiveAnalyticsView],
+      title: "Interop Views",
+      apps:
+        currentUser === USER_TRADER
+          ? [
+              {
+                ...reactiveTraderFxTradesView,
+                description:
+                  "FX Trades view - combine with Limit Checker view (Operations user)",
+                icons: [
+                  {
+                    src: `${WS_BASE_URL}/images/previews/interop-with-views-color-linking.png`,
```

Comment:
only way to get a "preview" on the middle row (don't ask me why) is to push something into the icon space .. works, but a bit squashed